_includes/templates/nav-about-us-media.html

```diff
@@ -0,0 +1,11 @@
+<a href="/the-bureau/">
+    <img src="../../static/img/nav-about-us-video.png"
```

Comment:
lol, no worries. .  @Scotchester, did you guys ever plan on implementing the 'sliding-nav_page-overlay' markup mentioned here: https://github.com/cfpb/cfgov-refresh/pull/151?